.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.4
+    - uses: actions/setup-go@v2.1.3
```

Comment:
Do we really need to pinpoint the exact versions? There is a "dynamic" tag named `v2` which points to the latest in version 2, but it will not be changed to v3 if it gets released in the future (so no breaking change).

Reply:
We don't need to used fixed versions, but it's useful if an action ever makes an unintended breaking change. My next step was to add dependabot to automate bumping of versions. That allows us to test new versions in our own workflows.

Reply:
ok, that sounds good to me. You can merge this PR.